ui/src/common/components/UnclickableTag/UnclickableTag.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Tag } from 'antd';
+import classnames from 'classnames';
+
+import './UnclickableTag.scss';
+
+class UnclickableTag extends Component {
+  render() {
+    const { children, className, ...otherProps } = this.props;
+
+    return (
+      <Tag
+        className={classnames('__UnclickableTag__', className)}
```

Comment:
```
className={`${className} __UnclickableTag__`}
```

would be better, `classnames` is a bit of an overkill when there is no conditional class to apply.

Reply:
I don't like it this way, because we should always have ``defaultProps`` and ``UnclickableTag`` always will have a ``className`` prop

```
    -       <UnclickableTag>
    +       <UnclickableTag
    +         className=""
    +       >
```